.github/scripts/ffmpeg/build.sh

```diff
@@ -25,7 +25,7 @@ if [ "$RUN_CONFIG" = 1 ] || [ ! -f "$FFMPEG_BUILD_PATH/Makefile" ] ; then
             --arch=aarch64 \
             --target-os=mingw32 \
             --cross-prefix=aarch64-w64-mingw32- \
-            $ADDITIONAL_OPTIONS
+            $HOST_OPTIONS
```

Comment:
What about TARGET_OPTIONS here?

Reply:
Depends on the point of view and what do you mean byt this comment in particular? From one point of view using `$TARGET_OPTIONS` instead of `$HOST_OPTIONS` here might seem more consistent with how it is used in the toolchain folder, from other point of view, host for target libraries equals target for the toolchain.